docs/developers_guide/Testing.md

```diff
@@ -35,27 +35,42 @@ val scenario =
 ```
 
 ### Creating test scenario runner
-Scenario should be executed inside a runner. `NuTestScenarioRunner` gives you another DLS for building runners.
+
+Scenario should be executed inside a runner. `TestScenarioRunner` gives you another DLS for building runners.
```

Comment:
We can add mention that sample components names are provided in `TestScenarioRunner` object

Reply:
I extended `Running scenario with data` section - I mentioned there about test sources and sinks. This one sentence doesn't look good:
> Results are collected using `sink` component in `liteBased` case and `invocationCollector` in `flinkBased` case
but it's showing current situation :/